### ISSUE TITLE
Maven Repository for prometheus-spring-boot-starter?

### ISSUE THREAD

**Issue by rohitghatol** [thomasdarimont/prometheus-spring-boot-starter#1]
Hi

First of all thanks for this open source project. I would love to give it a try. 

However, I am not able to pull the maven dependencies for this project. Which Maven Repositories are we keeping these in?

<dependency>
    <groupId>de.tdlabs</groupId>
    <artifactId>prometheus-spring-boot-starter</artifactId>
    <version>0.0.3-SNAPSHOT</version>
</dependency>

**Comment by dashaun** [comment]
I don't think this is in a central repo (yet).  You should just grab the project and build/deploy it to your own repo.
